src/tests/test_bias_report.py

```diff
@@ -56,7 +56,12 @@ def helper(input_filename, expected_filename, config_file):
             print('testing {} ...'.format(col))
 
             try:
-                if np.mean(combined_data[col + "_x"] - combined_data[col + "_y"]) > EPS:
+                # TypeError: numpy boolean subtract, the `-` operator, is
+                # deprecated, use the bitwise_xor, the `^` operator, or the
+                # logical_xor function instead.
+                # found online that casting as float64 will go around, but
+                # would like to get Jesse's take on best way to avoid issue.
+                if np.mean(combined_data[col + "_x"].astype("float64") - combined_data[col + "_y"].astype("float64")) > EPS:
```

Comment:
uhhh … yeah, i mean, what is this data? why are we subtracting a boolean from a boolean? do we _want_ to treat these things as numbers?